### DISCUSSION TITLE
Cannot find CefSharp.OutOfProcess in Nuget

### DISCUSSION THREAD

**Discussion by davidko1221** [cefsharp/CefSharp.OutOfProcess#25] [General]
Is CefSharp.OutOfProcess in a Nuget package? It doesn't appear in the Nuget search in Visual Studio 2022 (target framework 4.8), and the readme file in https://github.com/cefsharp/CefSharp.OutOfProcess does not indicate that OutOfProcess is even downloadable from Nuget. 

Thank you.

**Comment by amaitland** [comment]
> Is CefSharp.OutOfProcess in a Nuget package?

No. See #1
